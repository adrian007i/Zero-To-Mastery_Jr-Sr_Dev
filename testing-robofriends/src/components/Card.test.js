import {render, screen} from '@testing-library/react'
// import userEvent from '@testing-library/user-event' 
import '@testing-library/jest-dom'
import Header from './Header';

test('renders Card component', async () => {
    render(<Header url="/" />)
    await screen.findByRole('heading')

    // ASSERT
    expect(screen.getByRole('heading')).toHaveTextContent('RoboFriends')
});