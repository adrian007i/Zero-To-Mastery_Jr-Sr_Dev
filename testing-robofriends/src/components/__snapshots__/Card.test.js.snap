// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Card component 1`] = `
<h1
  class="f1 hi"
  id="hello"
>
  RoboFriends
</h1>
`;
